modin/pandas/test/dataframe/test_binary.py

```diff
@@ -151,6 +154,32 @@ def test_comparison(data, op, other):
     )
 
 
+@pytest.mark.skipif(
+    Engine.get() not in ("Ray", "Dask"),
+    reason="Modin on this engine doesn't create virtual partitions.",
+)
+@pytest.mark.parametrize(
+    "left_virtual,right_virtual", [(True, False), (False, True), (True, True)]
+)
```

Comment:
Maybe we should just parametrize `left_virtual` and `right_virtual` independently? That way we would also check both non-virtual left and right, which might be an extra test but still nice to have

Reply:
I think we shouldn't check non-virtual left and right. The ordinary binary operations test should definitely cover that, and I don't want to add a redundant test.